docs/simple-example.md

```diff
@@ -159,11 +87,46 @@ How `Secrets` and `ConfigMaps` are stored on disk mirror the storage for
 where a file is created for each key in a `ConfigMap` `data` map or `binaryData` map and each key in a `Secret`
 `data` map.
 
-If you want to try other `ConfigMaps` or a `Secret`, first clear out the existing application:
+## BuildConfig example
```

Comment:
This section is sufficiently complex that I would put it in a separate document.